qemu/tests/balloon_memhp.py

```diff
@@ -30,7 +30,7 @@ def check_memory():
         Check guest memory
         """
         if params['os_type'] == 'windows':
-            memhp_test.check_memory(vm)
+            memhp_test.check_memory(vm, wait_time=3)
```

Comment:
LGTM on tp-qemu part, but I don't think the VT patch also need this judgment, you could not ensure that linux never hit this issue and seems it's a double-check.

Reply:
Yes, update Avacado-vt code.Pls have a check.thx.